pyiron_atomistics/vasp/base.py

```diff
@@ -777,13 +817,20 @@ def get_final_structure_from_file(self, filename="CONTCAR"):
             pyiron.atomistics.structure.atoms.Atoms: The final structure
         """
         filename = posixpath.join(self.working_directory, filename)
+        # Backwards compatibility for old datasets using old vasp_sorter method
+        # DEPRECATE THIS WITH VASP_SORTER IN STRUCTURE.PY
+        try:
+            _idx_pyiron_to_user = self._idx_pyiron_to_user
+        except:
```

Comment:
```suggestion
        except AttributeError:
```

It's good form to be as targeted as possible when catching exceptions